src/meadowrun/deployment_manager.py

```diff
@@ -555,6 +602,21 @@ def _get_path_and_hash(
             file_to_hash = path_to_spec
         with open(file_to_hash, "rb") as spec:
             spec_contents_bytes = spec.read()
+
+        if environment_spec.editable_install:
+            # Additionally search for some extra files that influence how the resulting
+            # env is built, at least when it's using an editable install. If not, this
+            # may trigger an unnecessary rebuild in rare cases.
+
+            # TODO: search in other paths? This only works if these files live in the
+            # same place as the requirements file.
+            file_to_hash_folder = os.path.dirname(file_to_hash)
+            for extra_file in ("setup.py", "setup.cfg", "pyproject.toml"):
+                check = os.path.join(file_to_hash_folder, extra_file)
+                if os.path.exists(check):
+                    with open(check, "rb") as f:
+                        spec_contents_bytes += f.read()
```

Comment:
Right now this is getting passed into both `_get_environment_spec_prerequisites` and `_hash_spec`, is that intentional? Definitely needs to get passed into `_hash_spec`, haven't thought too hard about `_get_environment_spec_prerequisites`

Reply:
thanks, was not intentional, will fix.